ui/src/components/songsView/SongsTitleCard.tsx

```diff
@@ -0,0 +1,111 @@
+import { Container, Box } from '@mui/material';
+import { SongViewSchema } from '../../types/song.types';
+import Accordion from '@mui/material/Accordion';
+import AccordionSummary from '@mui/material/AccordionSummary';
+import AccordionDetails from '@mui/material/AccordionDetails';
+import Typography from '@mui/material/Typography';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import Grid from '@mui/material/Grid';
+import Chip from '@mui/material/Chip';
+import InfoIcon from '@mui/icons-material/Info';
+
+type SongTitleCardProps = {
+  song: SongViewSchema | undefined;
+};
+
+const SongsTitleCard = (props: SongTitleCardProps) => {
+  const song = props.song;
+
+  return (
+    <>
+      <Container>
```

Comment:
something is wrong with the width for mobile. 
<img width="390" alt="image" src="https://github.com/hmcc-global/psalted-2/assets/24607458/4ed1b8e5-a937-4259-8e4e-1d22d154f1aa">

Reply:
aduh more like i didnt even thought abt mobile at all HAHAHA